DoxygenRule.cmake

```diff
@@ -85,8 +85,8 @@ if(NOT DOXYGEN_CONFIG_FILE)
 endif()
 
 add_custom_target(doxygen_install_${PROJECT_NAME}
-  ${CMAKE_COMMAND} -DCMAKE_INSTALL_LOCAL_ONLY=ON
-  -P ${PROJECT_BINARY_DIR}/cmake_install.cmake
+  ${CMAKE_COMMAND}
+  -P ${PROJECT_BINARY_DIR}/${PROJECT_INCLUDE_NAME}/cmake_install.cmake
```

Comment:
Examples are not needed for generating documentation, are they? Only the headers are, so the minimal subset is installed which is also faster. It is not possible to install everything anyway because then it tries to also install binaries of subrojects, which have not been build (only subproject libs are built if you do make doxygen_PROJECTNAME, so failure).